catalog/admin-dto.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package catalog;
+
+import "catalog/domain.proto";
+
+option java_multiple_files = true;
+option java_package = "io.prodrink.catalog.generated.dto";
+option java_outer_classname = "DrinkDtoOuter";
+option csharp_namespace = "prodrink.catalog";
+
+message AdminServiceCallResponse {}
+
+message DeletionRequest {
+    int32 entity_id = 1;
+}
+
+message DrinkCreationRequest {
+    string name = 1;
+    int32 category_id = 2;
+    repeated int32 property_ids = 3;
+    repeated string image_urls = 4;
+    string description = 5;
+}
+
+message DrinkUpdateRequest {
+    int32 drink_id = 1;
+    string new_name = 2;
+    int32 new_category_id = 3;
+    repeated int32 new_property_ids = 4;
+    repeated string new_image_urls = 5;
+    string new_description = 6;
+}
+
+message CategoryCreationRequest {
+    string name = 1;
+    int32 parent_category_id = 2;
+    string image_url = 3;
+}
+
+message CategoryUpdateRequest {
```

Comment:
Name, image_url and parent_catogory_id (mby) fields are missing here.

Reply:
well, everything is missed here :thinking: